### PR TITLE
ADD argNorm version number to tsv outputs

### PR DESCRIPTION
**Pull request by Vedanth-Ramji**
The current argNorm version number is now added to tsv outputs under the 'argNorm_version' column to match with the way the software versions of ARG annotation tools are listed

### CODE REVIEW

**Comment by luispedro** [comment]
I don't like this format. I know hAMRonizer does it like this, so it could be fine for hamronized results, but not for the argNorm ones.

I would prefer it as a comment at the top of the file.

File: tests/test_normalizers.py [hunk 1]
```diff
@@ -9,15 +9,18 @@ def get_normed(normalizer, input_path):
     normed = normalizer.run(input_path)
     normed = normed.apply(lambda x: x.str.strip() if isinstance(x, str) else x).replace('', np.nan)
     return normed
- 
+
+def get_golden_file(path):
+    return pd.read_csv(path, sep='\t', skiprows=1)
```
Comment (luispedro) [inline tests/test_normalizers.py:14]: Instead of `skiprows=1`, it's better to do `comment='#'`

**Review by luispedro — CHANGES_REQUESTED** [review]

File: tests/test_normalizers.py (hunk 1 above)
Comment (Vedanth-Ramji) [reply tests/test_normalizers.py:14]: I just tried this right now, and I realized that a few ARG annotation tools use '#' as part of their column headers. For example here: https://github.com/BigDataBiology/argNorm/blob/main/outputs/raw/abricate.argannot.tsv, the file column header is given as '#FILE'. So would it be better to stick with `skiprows=1` as I know for sure that the argNorm version will only be on the first line?

Comment (luispedro) [reply tests/test_normalizers.py:14]: I see.

Just make sure this is well documented. In particular, in the release notes, there needs to be a bold warning type thing of "We are changing the output format and all your scripts will break!"

Comment (Vedanth-Ramji) [reply tests/test_normalizers.py:14]: Added this in #70

**Review by luispedro — APPROVED** [review]

**Pull request merged**